setup.py

```diff
@@ -6,7 +6,7 @@
     from distutils.core import setup
 
 setup(name='flexmock',
-      version='0.10.4',
+      version='0.10.5',
```

Comment:
This file is removed in the next PR right?

Reply:
Nope, I added it back because it is needed for Fedora builds. Setup.py is also needed for older pip versions so it might be good to keep it around for a bit longer.